_posts/2023-10-04-sep-2023-roundup.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "What we've been reading in September (2023)"
+author: tyler
+tags: [roundup]
+---
+
+<!-- excerpt start -->
+
+Here are the articles, videos, and tools that we've been excited about this
+September. 
+
+<!-- excerpt end -->
+
+We hope you enjoy these links, and we look forward to hearing what you've been
+reading in the comments or [on the Interrupt Slack](https://interrupt-slack.herokuapp.com/).
+
+
+
+## Articles & Learning
+
+- [**Memfault Webinar - An Empowering Endeavor: Predicting and Improving Device Battery Life**](https://hubs.la/Q023ppsY0)<br>
+I recently wrote a blog post about [monitoring battery life](https://interrupt.memfault.com/blog/monitoring-battery-life) and on Thursday, October 12th I'll be leading a webinar dedicated to predicting and optimizing battery performance for power-constrained devices. [Join us live at 8AM PT | 11AM ET | 5PM CET](https://hubs.la/Q023ppsY0) and bring the battery questions that drain you for our audience Q&A!
```

Comment:
I think I'd prefer to see this at the bottom with the other webinars like we discussed last time.

Reply:
a couple of counter-arguments I would make: 

- it's our only upcoming webinar (which currently drive the most pipeline out of all marketing channels)
- it's directly linked to tyler's battery post, so it's kind of a like a 2-for-1
- the roundups are technically under tyler's name, so I think it'd make sense to plug his own webinar

if you feel really strongly about it I can move it, but we are trying to more closely tie Memfault and Interrupt together, and this one doesn't feel too "asky" IMHO